Cargo.toml

```diff
@@ -512,7 +514,7 @@ sources-nats = ["dep:nats", "dep:nkeys"]
 sources-nginx_metrics = ["dep:nom"]
 sources-opentelemetry = ["dep:prost-types", "sources-http", "sources-utils-http", "sources-vector", "opentelemetry"]
 sources-postgresql_metrics = ["dep:postgres-openssl", "dep:tokio-postgres"]
-sources-prometheus = ["dep:prometheus-parser", "sinks-prometheus", "sources-http", "sources-utils-http"]
+sources-prometheus = ["dep:prometheus-parser", "sinks-prometheus", "sources-http", "sources-utils-http", "sources-http_scrape"]
```

Comment:
Would it make sense to put the common bits of these sources into the sources utils as well? It's a little expected to see another actual source being brought in here.

Reply:
I kind of anticipated someone suggesting moving the common bits (as I suggested in my initial comment for the PR, though in the context of independent crates for sources) , so yeah I kind of think it makes sense. It feels more natural to me to have the `prometheus_scrape` source not depend directly on another source as well. I just wasn't exactly sure the best place to put it.

Reply:
Addressed in https://github.com/vectordotdev/vector/pull/13793/commits/45396a944621db1a035122e368b4fb4b07c69066